cat/CaT-service.yaml

```diff
@@ -4691,13 +4950,17 @@ components:
       properties:
           id:
             type: string
+            description: ID for the document
           fileName: 
-            type: string  
+            type: string
+            description: The suggested filename for the document
```

Comment:
Isn't this just `the filename` (for the client's info)?

Reply:
Yes, they can use it to save the binary as the same name or a different name as they see fit.